lib/polling.js

```diff
@@ -48,46 +52,62 @@ TradeOfferManager.prototype.doPoll = function(doFullUpdate) {
 	}
 
 	this.emit('debug', 'Doing trade offer poll since ' + offersSince + (fullUpdate ? ' (full update)' : ''));
-	this.getOffers(fullUpdate ? EOfferFilter.All : EOfferFilter.ActiveOnly, new Date(offersSince * 1000), (err, sent, received) => {
-		if (err) {
-			this.emit('debug', "Error getting trade offers for poll: " + err.message);
-			this.emit('pollFailure', err);
+
+	var filter = fullUpdate ? EOfferFilter.All : EOfferFilter.ActiveOnly;
```

Comment:
Can't fight with that. Still, it saves line length, and naming such variables is a good practice (self-documenting code) in general.